src/atoms/title/AppTitle.ts

```diff
@@ -0,0 +1,29 @@
+import { TTitle } from '../../types';
+
+export function AppTitle({
+    tag = 'h1',
+    textContent = '',
+    innerHTML,
+    classes = [],
+}: TTitle): HTMLElement {
+    const headerTags: (keyof HTMLElementTagNameMap)[] = [
+        'h1',
+        'h2',
+        'h3',
+        'h4',
+        'h5',
+        'h6',
+    ];
+    const headerTag = headerTags.includes(tag) ? tag : 'h1';
```

Comment:
This can possibly cause an unexpected behavior, for example, for some reason I want to create title with `<a>`, and I pass 'a' to this component as a `tag` property, but this line converts it to just h1, I advise to use enum with list of allowed values as `tag` type.

Reply:
done